core/src/test/scala/unit/kafka/tools/ConsoleConsumerTest.scala

```diff
@@ -495,4 +514,76 @@ class ConsoleConsumerTest {
       Exit.resetExitProcedure()
     }
   }
+
+  /**
+    * There is no test for the default key and value deserializer properties
+    */
+  @Test(expected = classOf[ShouldExitException])
+  def shouldExitOnUnrecognisedProperty() {
+    val expectedMessage = Some("Unrecognised arguments ({not=ok, bad.property=not.good}) passed to kafka.tools.DefaultMessageFormatter.")
+    Exit.setExitProcedure((_, message) => {
+      assertEquals("Console consumer should exit with help information with bad properties listed", expectedMessage, message)
```

Comment:
It's more brittle to assert the full error message. Perhaps we can assert that something smaller is in the message and still get the same assertion (that the properties are listed). e.g assert it contains `not=ok` or `Unrecognised`